pwiz_tools/Skyline/Model/Lists/ListDef.cs

```diff
@@ -41,6 +42,10 @@ public ListDef(string name) : base(name)
             Properties = ImmutableList<AnnotationDef>.EMPTY;
         }
 
+        public override string AuditLogText { get { return null; } }
```

Comment:
This works since ListDef has some properties with track attributes, therefore the reflection based ToString for this object would return something like {Properties=[...], IdProperty="...", DisplayProperty="..."}. But if it would not have any properties, the reflection ToStringwould try to use the AuditLogText. Generally there shouldn't be any IAuditLogObjects with IsName=false and no sub properties. ListDef shouldn't be an IAuditLogObject in the first place, but of course it is since XmlNamedElement implements it...